server/routes/register.js

```diff
@@ -30,12 +28,13 @@ router.post('/register', async (req, res) => {
             await user.save();
             res.status(201).send(user);
```

Comment:
I usually send a JWT token in register as well, so the user can be logged in after register as well